building/index.rst

```diff
@@ -13,6 +15,7 @@ OP-TEE developer setup. So a first place to start looking is probably at the
 .. toctree::
    :maxdepth: 2
 
+   quick_start_qemu
```

Comment:
I've tried to keep the generated output sorted in alphabetic order and if possible also keep it sorted in the toctree.